bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/tools/WaitForTopicValue.java

```diff
@@ -46,11 +46,11 @@ public WaitForTopicValue(MqttBrokerConnection connection, String topic)
         final MqttMessageSubscriber mqttMessageSubscriber = (t, payload) -> {
             future.complete(new String(payload));
```

Comment:
The default charset varies from system to system so it is better to specify it yourself instead of using the default.